src/components/RestrictionTypeInput/RestrictionTypeInput.vue

```diff
@@ -0,0 +1,55 @@
+<template>
+    <FormRow>
+        <template v-slot:label
+            ><Tooltip word-wrap placement="top-start" :content="$t('form_label_restriction_type_tooltip')">
+                <Icon type="ios-information-circle-outline" />
+            </Tooltip>
+            {{ $t('type') }}:
+        </template>
+        <template v-slot:inputs>
+            <div class="inputs-container select-container">
+                <Select
+                    :value="value"
+                    :disabled="disabled"
+                    :placeholder="$t('type')"
+                    class="select-size select-style"
+                    @on-change="onBlockTypeChange"
+                >
+                    <Option key="Block" value="block">
+                        {{ $t('block') }}
+                    </Option>
+                    <Option key="Allow" value="allow">
+                        {{ $t('allow') }}
+                    </Option>
+                </Select>
+            </div>
+        </template>
+    </FormRow>
+</template>
+
+<script lang="ts">
+// extenal dependencies
+import { Component, Prop, Vue } from 'vue-property-decorator';
+
+// child components
+import FormRow from '@/components/FormRow/FormRow.vue';
+
+export enum RestrictionBlockType {
+    BLOCK = 'block',
```

Comment:
Not using the RestrictionFlag?

Reply:
It wasn't exported. Marked as @internal in the SDK.